core/extensions/impl/storage_extension.hpp

```diff
@@ -24,6 +24,10 @@ namespace kagome::extensions {
         std::shared_ptr<runtime::WasmMemory> memory,
         std::shared_ptr<storage::changes_trie::ChangesTracker> changes_tracker);
 
+    inline void reset() {
+      [[maybe_unused]] auto res = storage_provider_->rollbackTransaction();
```

Comment:
?

Reply:
rollbackTransaction returns outcome::result, which is required to be used as I understand

Reply:
Hm... Why shouldn't it be checked here, could you clarify?

Reply:
because result does not have value if there is no transaction, which is correct behaviour in that case

Reply:
But what if there is another error?